### PR TITLE
WebUI files_external delete mount

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
This PR webUI tests for files_external delete local storage mount.

Related issue:  #32852

## How Has This Been Tested?
🤖

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Acceptance test

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tests/acceptance/features/bootstrap/WebUIGeneralContext.php [hunk 1]
```diff
@@ -501,8 +501,8 @@ public function aFileWithSizeAndNameHasBeenCreatedLocally($size, $name) {
 
 	/**
 	 *
-	 * @When the user reloads the current page of the webUI
-	 * @Given the user has reloaded the current page of the webUI
+	 * @When the user/administrator reloads the current page of the webUI
+	 * @Given the user/administrator has reloaded the current page of the webUI
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUIGeneralContext.php]: This is a good thing - helps us write more readable tesst scenarios.

**Review by phil-davis — APPROVED** [review]
LGTM

File: tests/acceptance/features/webUIAdminSettings/adminStorageSettings.feature [hunk 2]
```diff
@@ -87,3 +87,52 @@ Feature: admin storage settings
...
+    Given user "user0" has been created with default attributes
+    And group "newgroup" has been created
+    And user "user0" has been added to group "newgroup"
+    And the administrator has browsed to the admin storage settings page
+    And the administrator has enabled the external storage
+    And the administrator has created the local storage mount "local_storage1" from the admin storage settings page
+    And the administrator has added group "newgroup" as the applicable group for the last local storage mount from the admin storage settings page
+    And group "newgroup" has been deleted
+    When the administrator reloads the current page of the webUI
+    Then the last created local storage mount should be listed on the webUI
```
[the remark this answers is not in the document]
Comment (paurakhsharma) [reply tests/acceptance/features/webUIAdminSettings/adminStorageSettings.feature]: Yes, newgroup is listed in the applicable list. And the status symbol is green as well. And I think this is an expected behavior.

**Comment by paurakhsharma** [comment]
Backport on: #33857

**Pull request merged**